### PR TITLE
Featured card re factor

### PR DESCRIPTION
**Pull request by andreykishtov**
redone all Featured Cards components with simple ant design ui.
it a first draft with minimal data(more when we will have real data to work with).

### CODE REVIEW

**Comment by andreykishtov** [comment]
i don`t think there should be images in featured and freshonsite because there are so small

**Comment by Alaev** [comment]
@andreykishtov I was asking for a screen shot of your work so I would know what I am reviewing. :D

File: src/components/FeaturedCard/components/Special.js [hunk 1]
```diff
@@ -4,8 +4,14 @@ import { Card } from 'antd';
 import 'antd/lib/card/style/css';
 
 const Special = ({ data }) => (
-  <Card title={data.title} extra={<a href="">More</a>} style={{ width: 300 }}>
-    <p>{data.text}</p>
+  <Card title={data.title} extra={<a href="">More</a>} bodyStyle={{ padding: 0 }} style={{ width: 300 }}>
```
Comment (Alaev) [inline src/components/FeaturedCard/components/Special.js]: @andreykishtov I was asking for screenshot. however if you added image to it that would be cool to see.
Can you add screenshot of before and after?

**Comment by andreykishtov** [comment]
screenshot from 2017-11-27 17-59-58

**Comment by andreykishtov** [comment]
featured

**Comment by andreykishtov** [comment]
featuredcarddiftrent

**Comment by Alaev** [comment]
@andreykishtov Also on click it will change the design of the right side. Please create the same component as we had with matirial

**Comment by andreykishtov** [comment]
I remade Special so it will use existing LocationCard.
redone Featured List and FreshOnSiteList.
made udjustments to gilis map card @gilbaadani .
there is a problem with styling with the map
```javascript
//mapWithMarkers File
containerElement: <div style={{ height: '350px' }} />,

//HomePage Styles styles should use props
export const Left = styled.div`
  min-height: 350px;
  max-height: 400px;
```

**Comment by andreykishtov** [comment]
special new

**Comment by andreykishtov** [comment]
timeline

**Comment by andreykishtov** [comment]
would be glad for some more pointers on how to style better

File: src/components/FeaturedCard/components/ListCards.styled.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+export const Wrapper = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+`;
+
+export const Image = styled.img`
+  //border-radius: 50%;
```
Comment (uvluni) [inline src/components/FeaturedCard/components/ListCards.styled.js]: Remove comment

**Pull request merged**